shared/fs/row/files-loading-hoc.js

```diff
@@ -7,15 +7,22 @@ import * as Types from '../../constants/types/fs'
 
 type OwnProps = {
   path: Types.Path,
+  destinationPickerIndex?: number,
```

Comment:
Is this used anywhere, or will this be a future PR? (I don't see anything in this one...)

Reply:
It's actually used below in `mapDispatchToProps` to detect if we are in the destination picker or just the main folder view.

Reply:
Just merged; let me know if you still have questions. Thanks for reviewing!